lib/Partitioner/Partitioner.cpp

```diff
@@ -16,6 +16,23 @@
 
 #include "glow/Partitioner/Partitioner.h"
 
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/raw_ostream.h"
+
+/// -log-partition - Command line option to dump Partitioner logs.
+static llvm::cl::OptionCategory PartitionerCat("Glow Partitioner Options");
+static llvm::cl::opt<bool>
```

Comment:
why not always log this info?

I'd be very interested in dumping this every time I run onnxifi.

Reply:
I think for release mode, we might not want such messages every time. Maybe this can be enabled automatically in debug mode? Another way I think user can change the default value during their test?

Reply:
I do not think there is any overhead with this logging for compile time. It's fine to log in release unless it's on the hot path.

Reply:
I just thought people may want to have some control to enable the logs :P